vm/devices/firmware/firmware_uefi/src/service/nvram/spec_services/mod.rs

```diff
@@ -180,7 +180,7 @@ where
 }
 
 #[derive(Clone, Copy, Debug, Protobuf, Inspect)]
-enum RuntimeState {
+pub enum RuntimeState {
```

Comment:
Why did this become pub?

Reply:
does this just need to be `pub(crate)`? or why is it pub at all?